go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/wolfi-dev/wolfictl
 
-go 1.23.4
+go 1.24.0
```

Comment:
should we require for others consume this to be on go1.24?

Reply:
yes! also next syft release will bump us to 1.24 anyway.